app/assets/stylesheets/runtime.scss

```diff
@@ -8,89 +8,67 @@
 // More thorough context can be found in: https://github.com/forem/rfcs/pull/27
 
 // Elements with all of these classes are not visible by default
-.ForemWebView,
-.ForemWebView-iOS,
-.ForemWebView-Android,
-.PWA,
-.PWA-iOS,
-.PWA-Android,
-.PWA-Windows,
-.PWA-Linux,
-.PWA-macOS,
-.Browser,
-.Browser-iOS,
-.Browser-Android,
-.Browser-macOS,
-.Browser-Windows,
-.Browser-Linux {
+.ForemWebView-only,
```

Comment:
This change was discussed here: 

https://github.com/forem/forem/pull/13662#discussion_r626903852